packages/grid/data-grid/package.json

```diff
@@ -38,15 +38,7 @@
     "tslib": "^2.0.0"
   },
   "devDependencies": {
-    "@material-ui/x-grid-modules": "^4.0.0-alpha.2",
-    "@rollup/plugin-node-resolve": "^8.0.1",
-    "rollup": "^2.6.1",
-    "rollup-plugin-cleaner": "^1.0.0",
-    "rollup-plugin-command": "^1.1.3",
-    "rollup-plugin-dts": "^1.4.7",
-    "rollup-plugin-sourcemaps": "^0.6.2",
-    "rollup-plugin-terser": "^5.3.0",
-    "rollup-plugin-typescript2": "^0.27.1"
+    "@material-ui/x-grid-modules": "^4.0.0-alpha.2"
```

Comment:
this package does not exist anymore.

Reply:
Definitely, it's outdated. 

<img width="430" alt="Capture d’écran 2020-09-25 à 11 44 34" src="https://user-images.githubusercontent.com/3165635/94252639-7f6d5a80-ff24-11ea-9d7a-744d87603ada.png">

Are you happy with pushing the effort forward https://github.com/mui-org/material-ui-x/pull/341#issuecomment-698453061?

Reply:
I think that yarn hoist the package automatically in workspaces or monorepos to root folder of the repo, unless pkgs contains different versions... So keeping the same version of pkgs is better as it avoids nested node_modules.
However removing the dev dependencies of each package is not that great as it makes it less obvious to understand what each package needs and what are their real dependencies. Also if we decide to move the package outside this repo then we would have to rebuild each package.json
Lastly in the root package.json we can easily end up with a bunch of dependencies that we won't know if they are being used by any of the nested packages.

Reply:
Note if we don't hoist, http://dependabot.com/ will update each very for each package individually;

Reply:
> less obvious to understand what each package needs and what are their real dependencies.

I think we can solve by having a single script command that all the packages use. It's what we do in the main repo and very likely what we want to do it here too so when we change something, we change it for all the packages at once. So I think it puts us in the right direction.

> if we decide to move the package outside this repo then we would have to rebuild each package.json

It would only happen if we move it to mui-org/material-ui, where the same packages should be available with the latest version. So, I think that it's a good thing, it will help catch inconsistencies.

> Lastly in the root package.json we can easily end up with a bunch of dependencies that we won't know if they are being used by any of the nested packages.

Probably true for any packages, there is only a tight link between the usage and its declaration.